src/igz_controls/components/more-info/more-info.component.js

```diff
@@ -36,18 +36,30 @@
                 isHtmlEnabled: false,
                 isDefaultTooltipEnabled: false,
                 defaultTooltipPlacement: 'auto',
-                defaultTooltipPopupDelay: '0',
+                defaultTooltipPopupDelay: '0'
             });
 
-            // Defaults trigger method to 'mouseenter' (hover)
-            ctrl.trigger = ctrl.trigger === 'click' ? 'click' : 'mouseenter';
-            ctrl.isDescriptionVisible = !isClickMode(); // need only for 'click' trigger. Init value - `false`
+            // Defaults trigger method to 'mouseenter'. Available 2 modes: `hover (mouseenter)` and `click`.
+            if (ctrl.trigger !== 'click') {
+                ctrl.trigger = 'mouseenter';
```

Comment:
move this to the `defaults` method above

Reply:
for what?